Havit.Blazor.Components.Web.Bootstrap/Grids/HxPager.razor.cs

```diff
@@ -118,6 +119,16 @@ static HxPager()
 	[Parameter] public string CssClass { get; set; }
 	protected string CssClassEffective => CssClass ?? GetSettings()?.CssClass ?? GetDefaults().CssClass;
 
+	protected override void OnParametersSet()
+	{
+		if (TotalPages < 1)
+		{
+			throw new InvalidOperationException($"{nameof(TotalPages)} has to be greater than or equal to 1.");
```

Comment:
Few commits ago we started to prefix "all" exceptions with a component name (except the exceptions regarding missing defaults). See the other `throw`s in the project.

Reply:
This still does not align with the new standard for exceptions
1. The component name should be taken from `this.GetType().Name` to reflect inheritance (if someone derives a new component from our component, we want the exception to report the real component name).
2. The typography is meant to be `[HxPager] Some exception text.`.

```csharp
throw new InvalidOperationException($"[{GetType().Name}] Some exception text...");
```